pkg/maputil/syncmap_test.go

```diff
@@ -0,0 +1,63 @@
+package maputil
+
+import (
+	"testing"
+
+	"github.com/stackrox/rox/pkg/sync"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSyncMapStoreLoad(t *testing.T) {
+	m := NewSyncMap[string, int]()
+	m.Store("a", 1)
+	m.Store("a", 2)
+	v, ok := m.Load("a")
+	assert.True(t, ok)
+	assert.Equal(t, 2, v)
+}
+
+func TestSyncMap(t *testing.T) {
+	wg := sync.WaitGroup{}
+	wg.Add(3)
+	m := NewSyncMap[string, int]()
+	go func() {
```

Comment:
What exactly do these tests verify?

Reply:
That there's no race (when run with `-race`) when accessing from different goroutines.